apps/editing-toolkit/editing-toolkit-plugin/coming-soon/styles.scss

```diff
@@ -0,0 +1,167 @@
+@import '~@automattic/typography/styles/fonts';
+
+html {
+	/* No admin bar nor marketing bar on this page */
```

Comment:
Why no admin bar tho? 🤔

Reply:
Seems like it was done in v1 => D44342-code

In v1 a logged-in user doesn't see the coming soon page.

Makes sense to show the admin bar for logged in users in v2 since they'll be able to view the selected page I guess.